taskcluster/ci/android-build/kind.yml

```diff
@@ -49,5 +49,3 @@ jobs:
       toolchain:
         - android-libs
         - desktop-linux-libs
-        - desktop-macos-libs
-        - desktop-win32-x86-64-libs
```

Comment:
Since this task doesn't build for windows or darwin, I don't see why it should have to download the pre-built libs for those platforms.